src/pages/landingpage.js

```diff
@@ -2,12 +2,14 @@ import React from "react";
 
 // components
 import Navbar from "../components/navbar/navbar";
+import Lpcontent from "../components/Lpcontent/Lpcontent.js";
 import Search from "../components/search/search";
 
 function LandingPage() {
     return (
         <React.Fragment>
             <Navbar/>
+            <Lpcontent/>
```

Comment:
code inside Lpcontent should be moved inside LandingPage.js. Lpcontent is not needed